src/plone/volto/locales/plone.volto.pot

```diff
@@ -37,8 +37,12 @@ msgstr ""
 msgid "Collections are migrated to Pages with Listing Blocks configured like the Collection."
 msgstr ""
 
+#: plone/volto/profiles.zcml:24
+msgid "Configures a new site with Volto and standard content types."
+msgstr ""
+
 #: plone/volto/profiles.zcml:16
-msgid "Configures the Plone backend to work with Volto, the new default frontend for Plone 6."
+msgid "Configures the Plone backend to work with Volto."
```

Comment:
We should avoid changing a `msgid`. It's a unique identifier for a `msgstr` for internal use only, and changing it may break translations that look for it. It's OK to update `msgstr`.

Reply:
The second part of the translations has been removed from the PO files, so it is not a breaking change.

Anyway, in this package the translation texts are mostly internal-admin related, so it is not as important as being public-facing strings.

So, I wouldn't do anything else related to this.